### ISSUE TITLE
Autolinking MindForger 1.51.0 Windows breaks view of heading 2

### ISSUE THREAD

**Issue by mseliger** [dvorka/mindforger#1216]
When I activate Autolinking with MindForger 1.5.1 then the heading of notes (heading 2) is not shown anymore in the title bar (Name) of the note, when I view my notes (not when I edit them).
Tested with / without the LivePreview.

**Comment by dvorka** [comment]
Thank you for the bug report Maria! 

TL:DR Please do not use autolinking yet.

Let me start with apology. I introduced a preview of autolinking feature a few versions back, however, it was slow and buggy. Therefore I removed autolinking from UI (menu) and kept it just in filesystem configuration. I did complete rewrite of autolinking for 1.50.0 (it's redesigned, significantly faster and it has much smarter parsing of Markdown). But I need more time to stabilize it (memory leaks, application instability, bugs similar to the one you reported...). Autolinking is not ready yet, but I'm really close. Please don't use autolinking and disable it: open `.mindforger.md` in your home directory and edit it to set `* Autolinking: no`. I believe that autolinking will be big MindForger feature one day - please stay tuned.

I disabled autolinking in Live preview to ensure performance, but when enabled, it is used in all previews.

**Comment by dvorka** [comment]
Reproducible `dev/1.52.0` (Coursera, CNN)

**Comment by dvorka** [comment]
Fixed 4fd155e3f3fe6fcb3eabf3ee93f185a3cc032c1b, will be released w/ MindForger 1.52.0
